src/commcare_cloud/ansible/roles/airflow/tasks/main.yml

```diff
@@ -10,6 +10,17 @@
     - "{{ airflow_home }}"
     - "{{ airflow_code_dir }}"
 
+- name: add deploy keys
```

Comment:
I don't think this is necessary since the keys are already added by the commcare role (which includes the airflow machines)

Reply:
added a new key

Reply:
this task is still not necessary - it's a duplicate of the one in commcarehq role which will already add the keys to the airflow machine.

I suppose if you haven't run the commcare role then the key won't exist so perhaps ok to duplicate - maybe we should pull it out into a separate file so we can reuse it (can do in new pr)